p2p/client_test.go

```diff
@@ -125,6 +125,58 @@ func TestGossiping(t *testing.T) {
 	wg.Wait()
 }
 
+func TestFraudProofGossip(t *testing.T) {
```

Comment:
I'm not sure it's useful to have this function that's just a copy of `TestGossiping` with `clients[4].GossipTx()` replaced with `clients[4].GossipFraudProof()`